client/lib/geocoding/test/index.js

```diff
@@ -15,6 +16,8 @@ import { geocode } from '../';
 const TEST_ADDRESS = '1600 Amphitheatre Parkway, Mountain View, CA';
 
 describe( 'geocoding', () => {
+	useNock();
```

Comment:
>Then wouldn't have to import `nock` explicitly in this file.

I actually like the explicitness, especially since it's really only the difference of a single import statement. I tend to find the implicitness of conventions to be a bit confusing, and find myself referring to docs more than I'd like to. The brevity of `useNock`, `useFakeDom`, etc. also provide a more obvious at-a-glance view of which helpers the test suite makes use of.